packaging/debian/control

```diff
@@ -28,7 +28,7 @@ Build-Depends: debhelper (>= 10~),
                librabbitmq-dev,
                python <sng-python2>, python-dev <sng-python2>, python-setuptools <sng-python2>,
                python3, python3-dev, dh-python,
-               python3-nose, python3-ply, python3-setuptools,
+               python3-ply, python3-setuptools,
```

Comment:
It's not a replacement, it's a removal.
Shouldn't we _replace_ `python3-nose` with `python3-pytest`?

Reply:
First of all if we really have to we can add it of course. 
But in my opinion it depends on deb builder environment (machine):
- if the deb build process requires to run `make-check` with full feature set (with all tests) yes, than this package (`python3-pytest`) will be needed (of course if the pip packages does not work -> for this there is a `requirements.txt` in root dir where pytest is also listed). 
- if the deb build process requires to run `make-check` but without full feature set (I have added a check before the pytest tests runs, if the pytest binary exists or not, the tests are run only if the pytest binary exist), than the `python3-pytest` package will not be needed.

Reply:
We should also consider that apt packages are way back than pip packages in package versions. And in daily basis (local dev or in dbld) we are using newest pytest packages (from pip). There can be feature differents in apt vs pip packages which could hurt later.